bwt/example_test.go

```diff
@@ -0,0 +1,70 @@
+package bwt_test
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/bebop/poly/bwt"
+	"golang.org/x/exp/slices"
+)
+
```

Comment:
should have an Example_basic test here showing how to use the package's basic functionality end-to-end

Reply:
Updated https://github.com/bebop/poly/pull/411/commits/1cc37557ed1da0cd9b1ef350466d5844ef375f11